test/api_compatibility.js

```diff
@@ -55,45 +81,59 @@ module.exports = function apiCompatibilityChecks(_createTracer) {
                 });
```

Comment:
last 3 checks are validating `extract` method, not `inject` as stated in L76.  Maybe move them to a separate `describe('extract')`?  Also extract is only called with binary format, we could add textmap.